docs/sql/statements/select.rst

```diff
@@ -159,7 +159,7 @@ Joined relation
 A ``joined_relation`` is a relation which joins two relations together. See
 :ref:`sql_dql_joins` ::
 
-    relation { , | join_type JOIN } relation [ ON join_condition ]
+    relation { , | join_type JOIN } relation {[ ON join_condition ] | [USING (col_names)]}
```

Comment:
`ON` and `USING` should be optional, so not sure if it is correct to put them into curly braces. Maybe something like this would work?
```suggestion
    relation { , | join_type JOIN } relation [ ON join_condition | USING (column_ident [, ...]) ]
```